lib/ex_unit/test/ex_unit/filters_test.exs

```diff
@@ -5,6 +5,106 @@ defmodule ExUnit.FiltersTest do
 
   doctest ExUnit.Filters
 
+  setup do
+    [
+      one: %{
+        one: true,
+        numeric: true,
+        test: :"test one",
+        test_type: :test
+      },
+      two: %{
+        skip: true,
+        numeric: true,
+        test: :"test two",
+        test_type: :test,
+        two: true
+      }
+    ]
+  end
+
+  @doc """
+  Evaluates thew filter precedence.
+  Tests are excluded, then included, and if they have
+  The following setting is given:
+
+      @tag :one
+      test "one", do assert(true)
+
+      @tag :skip
+      @tag :two
+      test "two", do assert(true)
+
+  """
```

Comment:
I think we don't need this comment, because it is duplicating what is written in the docs for the function.

Reply:
removed